src/Core/MakeModelMenu.elm

```diff
@@ -0,0 +1,355 @@
+module Core.MakeModelMenu exposing (init, update, subscriptions, view, Flags, Model, Msg)
+
+{-| Make/Model menu component
+
+
+# Exported
+
+@docs init, update, subscriptions, view, Flags, Model, Msg
+
+-}
+
+import CarwowTheme.Icons exposing (icon)
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import Html.Events exposing (onClick)
+import Core.Modal exposing (..)
+import CarwowTheme.Modal exposing (ModalProperties, PaddingStyle)
+import RemoteData exposing (RemoteData, WebData, sendRequest, update)
+import Http exposing (get)
+import Erl exposing (..)
+import Core.Data.Make exposing (..)
+import Core.Data.Model exposing (..)
+import Regex exposing (..)
+
+
+{-| The program which describes the MakeModelMenu, takes flags
+-}
+main : Program Flags Model Msg
```

Comment:
I don't think we should have this, we'll always need to re-implement it to get ports working, see this:

https://github.com/carwow/quotes_site/blob/master/app/elm/NotificationMenu.elm